config/Config-kernel.in

```diff
@@ -1418,8 +1418,7 @@ config KERNEL_JFFS2_FS_SECURITY
 
 config KERNEL_WERROR
 	bool "Compile the kernel with warnings as errors"
-	default BUILDBOT
-	default y if GCC_USE_VERSION_12
```

Comment:
Yes, this is unfortunately a pain point, I don't know how to solve better. Thus if we bump the default GCC version, then we should bump a version here. This is indeed another bug, you've just discovered, sorry.

Reply:
https://github.com/openwrt/openwrt/pull/15068/commits/fab97cbf4d7b81de99d60451d65aa9c9b687b1a4